documentation/Quickstart-Clear-Containers-in-a-Ubuntu-VM.md

```diff
@@ -55,11 +55,11 @@ All you need to do to use *ciao-down* on your machine is:
 
 
    ```
-   $ GOPATH/bin/ciao-down prepare -vmtype clearcontainers
-   $ GOPATH/bin/ciao-down connect
+   $ $GOPATH/bin/ciao-down prepare -vmtype clearcontainers
+   $ $GOPATH/bin/ciao-down connect
    ```
 
-## Usage of ciao-down:
+## Usage of ``ciao-down``
```

Comment:
Again, I think the change you are suggesting is structural - my change is simply to make `ciao-down` be rendered in a fixed-font (since it is a command). I removed the semi-colon in passing as it is erroneous for a heading in my opinion.